### DISCUSSION TITLE
Modifying Metamask UI

### DISCUSSION THREAD

**Discussion by PTS3000** [MetaMask/snaps#1732] [Q&A]
Hey everyone. In the snaps user interface section of the documentation, it says that a snap can modify the MetaMask UI using custom UI. So we can create prompts, alerts and confirmations with the snap_dialog. 
My question is, can we modify the Metamask UI in any other way for now? 
(For example, I would like to create a custom button, to use my snap)

Or is the "custom UI" the only way at the moment we can modify the UI?
Cheers, and thank you for helping!

**Comment by kpachhai** [comment]
No other way right now but I think there are plans to have more complex UI that can be customized. Not sure when though

**Comment by Montoya** [comment]
We are documenting upcoming UI components in [SIP-7](https://metamask.github.io/SIPs/SIPS/sip-7), including a planned button component
